### ISSUE TITLE
Global yamllint config file is not loaded

### ISSUE THREAD

**Issue by ggustafsson** [ansible-community/ansible-lint#1968]
##### Summary

ansible-lint does not detect and load global yamllint config file (e.g $XDG_CONFIG_HOME/yamllint/config which defaults to ~/.config/yamllint/config).
https://yamllint.readthedocs.io/en/stable/configuration.html

It seems ansible-lint is not looking for a global yamllint configuration file.

src/ansiblelint/yaml_utils.py:62:
```
    for file in [".yamllint", ".yamllint.yaml", ".yamllint.yml"]:
```
I'm not sure if just appending the global file paths to this line will fix it or not. I will leave that to someone who is more familiar with the source code.

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
$ ansible --version
ansible [core 2.12.2]
  config file = /Users/GLEG/Projects/Ansible/ansible.cfg
  configured module search path = ['/Users/GLEG/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.9/site-packages/ansible
  ansible collection location = /Users/GLEG/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/local/bin/ansible
  python version = 3.9.10 (main, Jan 15 2022, 11:48:00) [Clang 13.0.0 (clang-1300.0.29.3)]
  jinja version = 3.0.3
  libyaml = True

$ ansible-lint --version
ansible-lint 5.4.0 using ansible 2.12.2
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### OS / ENVIRONMENT

macOS

##### STEPS TO REPRODUCE

Create global yamllint config file at location "~/.config/yamllint/config". yamllint will pick it up but ansible-lint won't.

```console
$ cat ~/.config/yamllint/config
extends: default

rules:
  line-length: disable
```

##### Desired Behavior

ansible-lint should detect and load the same configuration files as yamllint.

```console
$ yamllint .
$
```

##### Actual Behavior

ansible-lint does not detect and load global yamllint configuration file.

```console (paste below)
$ ansible-lint          
WARNING  Listing 4 violation(s) that are fatal
yaml: line too long (167 > 160 characters) (line-length)
roles/macos/tasks/main.yml:168

yaml: line too long (278 > 160 characters) (line-length)
roles/macos/tasks/main.yml:169

yaml: line too long (277 > 160 characters) (line-length)
roles/macos/tasks/main.yml:170

yaml: line too long (222 > 160 characters) (line-length)
roles/macos/tasks/main.yml:171

You can skip specific rules or tags by adding them to your configuration file:
# .ansible-lint
warn_list:  # or 'skip_list' to silence them completely
  - yaml  # Violations reported by yamllint

Finished with 4 failure(s), 0 warning(s) on 42 files.
```

**Comment by ssbarnea** [comment]
We have no plans to use global yamllint config, only the in-repo configuration. That is by design because our defaults are different than yamllint project.
